nuxeo-core/nuxeo-core/src/main/resources/OSGI-INF/CoreExtensions.xml

```diff
@@ -31,6 +31,9 @@
     <schema name="collectionMember" src="schema/collectionMember.xsd"
       prefix="collectionMember" isVersionWritable="true"/>
 
+    <property schema="dublincore" name="creator" secured="true" />
+    <property schema="dublincore" name="contributors" secured="true" />
+    <property schema="dublincore" name="lastContributor" secured="true" />
```

Comment:
Do we plan to also secure `created` and `modified`?

Reply:
Good question, [NXP-27137](https://jira.nuxeo.com/browse/NXP-27137) doesn't state about them. I have no clue about secure them or not. Maybe @efge does?

Reply:
Yep, though Alain mentioned it in https://jira.nuxeo.com/browse/NXP-27137?focusedCommentId=387478&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-387478